package.json

```diff
@@ -112,19 +111,20 @@
     "@esbuild-plugins/node-modules-polyfill": "^0.1.4",
     "@fal-works/esbuild-plugin-global-externals": "^2.1.2",
     "@juggle/resize-observer": "^3.3.1",
+    "@rollup/plugin-inject": "^4.0.4",
     "@sage-bionetworks/esbuild-plugin-svgr": "^0.0.4",
     "@storybook/addon-actions": "^6.5.9",
     "@storybook/addon-essentials": "^6.5.9",
     "@storybook/addon-interactions": "^6.5.9",
     "@storybook/addon-links": "^6.5.9",
-    "@storybook/builder-vite": "^0.1.37",
+    "@storybook/builder-vite": "0.2.2",
     "@storybook/react": "^6.5.9",
     "@storybook/testing-library": "^0.0.13",
-    "@testing-library/dom": "^8.1.0",
-    "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^12.0.0",
-    "@testing-library/react-hooks": "^7.0.1",
-    "@testing-library/user-event": "13.0.16",
+    "@testing-library/dom": "^8.17.1",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.3.0",
+    "@testing-library/react-hooks": "^8.0.1",
+    "@testing-library/user-event": "^14.4.3",
```

Comment:
@testing-library/react v13 and @testing-library/user-event v14 caused most of the required changes.